public/Query.js

```diff
@@ -769,4 +771,31 @@ Query.executeFromSearch = function () {
 
 window.addEventListener('popstate', function () {
 	Query.executeFromSearch()
-})
\ No newline at end of file
+})
+
+/* Make table header fixed after scroll limit is reached */
+function fixedHeader() {
+
+	document.getElementById('fixed-table-header').innerHTML = ''
+
+	var headers = document.getElementsByClassName('header')
+	headers = Array.prototype.slice.call(headers)
+	var newTable = document.createElement('table')
+	var newTBody = document.createElement('tbody')
+
+	headers.forEach(function (each) {
```

Comment:
Why `Array.prototype.slice.call(headers)` and then `headers.forEach`? Why not `Array.prototype.forEach.call(headers, fn)`?